aws/resource_aws_instance.go

```diff
@@ -287,7 +287,12 @@ func resourceAwsInstance() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-
+			"host_id": {
```

Comment:
Two items:
* Can you please add documentation for this new argument in `website/docs/r/instance.html.markdown`?
* Can you please add a covering acceptance test in `aws/resource_aws_instance_test.go`?

Reply:
added documentation and test. I copied the test from the tenancy field, as I have done with most of the code when adding this.